webapp/src/Controller/API/JudgehostController.php

```diff
@@ -626,6 +631,7 @@ public function addJudgingRunAction(
         $outputDiff   = $request->request->get('output_diff');
         $outputError  = $request->request->get('output_error');
         $outputSystem = $request->request->get('output_system');
+        $teamMessage  = $request->request->get('team_message');
```

Comment:
If you do this, you can drop the null check
```suggestion
        $teamMessage  = $request->request->get('team_message', '');
```